android/src/main/kotlin/com/mytiki/sdk/capture/receipt/capacitor/AccountTypeEnum.kt

```diff
@@ -0,0 +1,77 @@
+package com.mytiki.sdk.capture.receipt.capacitor
```

Comment:
Add the copyright header to all files.

/*
 * Copyright (c) TIKI Inc.
 * MIT license. See LICENSE file in root directory.
 */